go.mod

```diff
@@ -51,3 +51,5 @@ require (
 	google.golang.org/grpc v1.22.1
 	gopkg.in/yaml.v2 v2.2.2
 )
+
+go 1.13
```

Comment:
I think you should delete this block , No description about this block from golang.org ,I don’t know how will it use in the future

Reply:
ok, i will remove it. This block added by my goland...😅

Reply:
done